Libraries/Hotcakes.Shipping.UpsFreight/PackingTypes.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
```

Comment:
This is missing the open source license header

Reply:
done. added license header.